apps/extension/src/core/llm/model.ts

```diff
@@ -345,7 +344,7 @@ export class Model {
     return {
       Authorization: opts.apiKey ? `${authPrefix}${opts.apiKey}` : undefined,
       "X-API-KEY": opts.apiKey || undefined,
-      "HTTP-Referer": opts.origin
+      "HTTP-Referer": "__no_txn_origin__",
```

Comment:
why did you change this? we do need to know the origin of transactions coming in

Reply:
mistake, was testing code. fixed.